src/containers/CodesplainAppBar.jsx

```diff
@@ -49,6 +57,11 @@ export class CodesplainAppBar extends React.Component {
     location.reload();
   }
 
+  handleSnippetSelected(key) {
+    const username  = cookie.load('username');
+    window.location = `/${username}/${key}`;
```

Comment:
I tried removing that line, and doing so causes the app to no longer actually load the selected snippet, despite updating the URL. I'm happy to remove the line, but I will need an alternative way to load the snippet if I do.